streams/in.m3u

```diff
@@ -394,6 +412,8 @@ http://103.199.161.254/Content/manoramanews/Live/Channel(ManoramaNews)/index.m3u
 http://103.199.161.254/Content/mathrubhuminews/Live/Channel(Mathrubhuminews)/index.m3u8
 #EXTINF:-1 tvg-id="MathrubhumiNews.in",Mathrubhumi News (1080p) [Not 24/7]
 https://ythls.onrender.com/channel/UCwXrBBZnIh2ER4lal6WbAHw.m3u8
+#EXTINF:-1 tvg-id="Mayyazhi.in",Mayyazhi (720p)
```

Comment:
Either not 24/7 or Geo-blocker.

Reply:
Accessible from Europe (VPN). So, not Geo-Blocked. 

![image](https://github.com/iptv-org/iptv/assets/61133303/1415568c-9310-4bbd-926c-849b8abf5539)